### PR TITLE
Feat: Add #481

### PR DESCRIPTION
**Pull request by vijayasantham12**
Hi @fyears, I have implemented this #481 feature but I didn't change manual sync notice message because I think manual sync notice message is not bad.

And @fyears please change zh_cn.json & zh_tw.json according to en.json for statusbar messages

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1180,8 +1191,10 @@ export default class RemotelySavePlugin extends Plugin {
         timeText = t("statusbar_time_hours", { time: hours });
       } else if (minutes > 0) {
         timeText = t("statusbar_time_minutes", { time: minutes });
-      } else {
+      } else if (seconds > 30) {
```
Comment (ZeroEkkusu) [inline src/main.ts:1194]: Thanks @vijayasantham12!

Sync On Save has options 1s, 10s, and 1m, so using 10s here would be perfect:
```suggestion
      } else if (seconds > 10) {
```

Comment (vijayasantham12) [reply src/main.ts:1194]: https://github.com/remotely-save/remotely-save/blob/286c37048647ea73f458abc6cf5e1e9c13e55a0a/src/main.ts#L769-L777

@ZeroEkkusu Thanks for the suggestion but as you can see, the sync statusbar message is only updated every 30 secs, so `> 10` has no effect

Comment (ZeroEkkusu) [reply src/main.ts:1194]: > the sync statusbar message is only updated every 30 secs

Do you think it's energy consuming to change to 10 seconds to align the intervals?

Comment (fyears) [reply src/main.ts:1194]: I think it's not necessary. I am now merging the pr as is. Thanks everyone.

**Pull request merged**